test/test_mechanize_pluggable_parser.rb

```diff
@@ -36,7 +36,16 @@ def test_parser
   def test_parser_mime
     @pp['image/png'] = :png
 
-    assert_equal :png, @pp.parser('x-image/x-png')
+    if Gem::Version.new(MIME::Types::VERSION) < Gem::Version.new('3.0')
+      # This is no longer true under mime-types 3 because the x- prefix has
+      # been deprecated by IANA. It is no longer safe to say that x-png and png
+      # are the same.
+      assert_equal :png, @pp.parser('x-image/x-png')
+    else
+      assert_equal Mechanize::File, @pp.parser('x-image/x-png')
+      @pp['x-image/x-png'] = :png
+      assert_equal :png, @pp.parser('x-image/x-png')
```

Comment:
forgive my ignorance... but is the point of the test to determine the equivalency of `x-image/x-png` and `image/png`? Or just to test that it's hooked up and we get _anything_ back? Can we drop the `x-png` part entirely to keep the test clean?